src/external/lib/view-engine/filters/ngr-points.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Creates extraction point/s description string based on number of grid reference points
+ * @param  {Object} points contains details of extraction point, including 1, 2 or 4 grid references
+ * @return {String}        description of extraction points
+ */
+const ngrPoints = points => {
```

Comment:
This feels like it could be neatly moved to a Nunjucks macro to display a single NGR because it is presentational.
I think we probably only need a filter for `formatGridReference`.

Reply:
I realised that this whole filter was a duplicate of a macro that was already created, so I will be deleting it and using the macro instead